### PR TITLE
added login/signup logic into Header component. Probably needs refact…

### LINKED ISSUE #4: Link login/sign up page to landing page

### PR DESCRIPTION
**Pull request by LemonFace0309**
…oring into a global state object.

- Copy and pasted from previous login modal page. Will probably move auth handling to atoms with Recoil next sprint.
- Spinner is currently in `Header` component instead of `landing/index.js` since I plan to move the `loading` state into Recoil later. Will refactor to move spinner out then too.

closes #4  

image
image

### CODE REVIEW

File: components/Auth/AuthDialog.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+}) => {
+  return (
+    <Dialog
+      fullWidth={true}
+      maxWidth="lg"
+      open={modalOpen}
+      onClose={() => setModalOpen(false)}
+      aria-labelledby="Login/Signup Modal"
+      className={styles.Dialogue}>
+      <Grid container className="my-8" alignItems="center" direction="row">
```
Comment (mabelene) [inline components/Auth/AuthDialog.js:38]: image
future task: make responsive, has a vertical scroll on my screen at the moment (prob bc my laptop screen is smaller ;-;)

**Review by mabelene — APPROVED** [review]

File: components/Sidebar.js [hunk 2]
```diff
@@ -1,99 +0,0 @@
-import React from "react";
```
Comment (mabelene) [inline components/Sidebar.js:1]: Did you accidentally delete this file

**Comment by mabelene** [comment]
looks really pretty!! good job

**Review by jimmyygit — APPROVED** [review]
nice

File: components/Auth/AuthDialog.js (hunk 1 above)
Comment (LemonFace0309) [reply components/Auth/AuthDialog.js:38]: thanks. Didn't take that into account.

File: components/Sidebar.js (hunk 2 above)
Comment (LemonFace0309) [reply components/Sidebar.js:1]: You don't need it in next

**Pull request merged**